presto-tests/src/main/java/io/prestosql/tests/H2QueryRunner.java

```diff
@@ -260,6 +263,15 @@ else if (DOUBLE.equals(type)) {
                             row.add(doubleValue);
                         }
                     }
+                    else if (JSON.equals(type)) {
+                        String stringValue = JsonFunctions.jsonParse(utf8Slice(resultSet.getString(i))).toStringUtf8();
```

Comment:
`JsonFunctions.jsonParse(utf8Slice(` should be called only after you know that `!resultSet.wasNull()`

Reply:
done